portfolio/pom.xml

```diff
@@ -35,7 +35,7 @@
         <version>2.2.0</version>
         <configuration>
           <!-- TODO: set project ID. -->
-          <deploy.projectId>YOUR_PROJECT_ID_HERE</deploy.projectId>
+          <deploy.projectId>amatasanchez-sps-summer20</deploy.projectId>
           <deploy.version>1</deploy.version>
         </configuration>
       </plugin>
```

Comment:
The changes made for the pull request should reflect the title. So in this case, all the commits should relate to updating the README file. The other commits made should be placed in a separate pull request under a title summarizing those changes.

Reply:
I renamed the pull request for now. Is it possible to create multiple pull requests for a branch or should I move the changes to another branch next time?

Reply:
I think the best way to go about it is to create a branch of this branch and make your changes there. Wait for the first branch's pull request to go through and then merge changes into the second branch. Then, create a pull request of the second branch. Ideally, the pull request should be short so that the review time doesn't take too long :)

Reply:
I agree. Multiple branches are the simplest way to keep the work parallelized, but it can be a little confusing and even tricky when the merge conflict appears. So if you feel that the flow isn't clear just reach us and we can share some tips.

Ideally this PR, should be 2 split into 2: 
- One PR that only contains the README.md . Title suggestion: "Update the README.md file to describe the repo."
- One other with the basic structure, explaining in the description exactly what is being changed.  e.g. just adding the unchanged project skeleton or replacing all the string by something, etc

I'm not requesting to split this into 2 different PR at this point, but I'm sharing with you how you could do that, since it's possible that you'll need that in the future.

There is multiples ways to split an already existing PR in multiple ones, but I suggest:
1- git cherry-pick -n (cherry-pick changes from commit without committing them yet)
2- git reset HEAD (unstage everything)
3- git add [file_path] (stage the specified files)
4- git clean -f (remove all untracked files, the ones not selected in the previous step)
5- git commit